Sergio/HW4/debug.md

```diff
@@ -0,0 +1,9 @@
+lines 2 and 6 would need to be altered to include the folder path.
+
+the lines should look like this afterwards.
+
+
+arguments = input/p098_words.txt
```

Comment:
The arguments line needs not to be changed as the 'transfer_inout_files line transfers the p098_words. txt into the scratch directory of the 'execution point'. It doesn't transfer the input folder just the file.